app-misc/piper/piper-0.4.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7} )
+
+inherit meson python-single-r1 udev
+
+DESCRIPTION="GTK configuration application for libratbag"
+HOMEPAGE="https://github.com/libratbag/piper"
+SRC_URI="https://github.com/libratbag/piper/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE=""
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+BDEPEND="
+	virtual/pkgconfig
+"
+RDEPEND="
+	${PYTHON_DEPS}
+	dev-libs/libratbag
```

Comment:
https://github.com/libratbag/piper/blob/0ab5c63ed404f619dcd6c380d8881d3f9eb3d406/meson.build#L16-L17 says that `>=dev-libs/libratbag-0.13` is needed.

Reply:
Correct, fixed.